test/extensions/filters/http/lua/lua_integration_test.cc

```diff
@@ -893,7 +893,7 @@ TEST_P(LuaIntegrationTest, BasicTestOfLuaPerRoute) {
 // Test whether Rds can correctly deliver LuaPerRoute configuration.
 TEST_P(LuaIntegrationTest, RdsTestOfLuaPerRoute) {
 // When the route configuration is updated dynamically via RDS and the configuration contains an
-// inline Lua code, Envoy may call lua_open in multiple threads to create new lua_State objects.
+// inline Lua code, Envoy may call luaL_newstate in multiple threads to create new lua_State objects.
```

Comment:
Thanks. @spacewander could you add a link to `luaL_newstate` ref here? Thank you!

Reply:
Sorry, I think you need to write it as: 

```
`luaL_newstate`
```

instead of lua_newstate. to skip adding newstate to dictionary.